<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="pkg_name" value="tof_vehicle_detection"/>
    <arg name="node_name" default="tof_vehicle_avoidance_control_node"/>
    <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." />

    <group ns="$(arg veh)">
        <remap from="$(arg node_name)/car_cmd" to="vehicle_avoidance_control_node/car_cmd"/>
        <remap from="$(arg node_name)/car_cmd_in" to="lane_controller_node/car_cmd" />
        <node  name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" required="true">
			<rosparam command="load" file="$(find tof_vehicle_detection)/config/$(arg node_name)/$(arg param_file_name).yaml"/>
        </node>
    </group>

    <!-- Publications -->

    <!-- Subscriptions -->

</launch>
